biz.aQute.bndlib/src/aQute/bnd/osgi/Macro.java

```diff
@@ -1617,6 +1617,15 @@ public String _basenameext(String[] args) throws Exception {
 		return result;
 	}
 
+	static final String _bndversionHelp = "${bndversion}, returns the currently running bnd version";
+
+	public String _bndversion(String[] args) throws Exception {
+		try (Analyzer a = new Analyzer()) {
+			String v = a.getBndVersion();
```

Comment:
You don't need to make an Analyzer. Just return `About.CURRENT.toString()`.

Reply:
I've just realized that myself. 

I wonder why the analizer is using this mechnism with the `bnd.info` file to determine the version. It looks to me as if the whole file and the mechanisms around it are obsolete.

Reply:
Probably :-)